alchemist/alchemist-euclidean-geometry/src/test/kotlin/it/unibo/alchemist/test/TestEuclidean2DShapeFactory.kt

```diff
@@ -11,6 +11,15 @@ import it.unibo.alchemist.model.interfaces.geometry.euclidean2d.Euclidean2DShape
 import it.unibo.alchemist.model.interfaces.geometry.euclidean2d.Euclidean2DShapeFactory
 import it.unibo.alchemist.model.interfaces.geometry.euclidean2d.Euclidean2DTransformation
 
+internal fun Euclidean2DShapeFactory.oneOfEachWithSize(size: Double) = mapOf(
+    "circle" to circle(size * 2),
+    "circleSector" to circleSector(size * 2, Math.PI, 0.0),
+    "rectangle" to rectangle(size, size),
+    "adimensional" to adimensional()
+)
+
+internal const val DEFAULT_SHAPE_SIZE: Double = 1.0
```

Comment:
needs refactoring. Maybe public in the companion? Why is this top level?

Reply:
It's widely used. Moved to the companion object of `TestEuclidean2DShapeFactory`.